app/assets/stylesheets/notifications-welcome-broadcast.scss

```diff
@@ -0,0 +1,20 @@
+// TODO: [@thepracticaldev/delightful]: Refactor nesting in this file
+// in addition to notifications.scss so that the nesting is
+// not as deep as it currently is
```

Comment:
Thanks for adding this comment! 👍 I definitely prefer not to have to nest like this, but I think it makes sense to extract the relevant styles into a separate stylesheet so that you can make changes without having to worry about breaking other notification styles, and then come back to un-nesting these later.